src/main/java/core/basesyntax/service/operation/OperationHandler.java

```
@@ -0,0 +1,7 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.model.FruitTransaction;
+
+public interface OperationHandler {
+    void apply(FruitTransaction transaction);
```

Add explicit Javadoc on the `apply` method to document expected contract: the `transaction` parameter must be non-null and contain valid data (non-null operation, non-empty fruit, non-negative quantity). Specify that implementations must validate inputs and throw informative `RuntimeException` on invalid input rather than swallowing exceptions. This aligns the interface with the checklist requirement to validate input data and not ignore exceptions.